yaptide/static/openapi.yaml

```diff
@@ -299,14 +302,65 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/BasicResponse'
+    get:
+      description: Allows for getting job status.
+      parameters:
+        - in: query
+          name: job_id
+          schema:
+            type: string
+      responses:
+        '200':
+          description: TODO
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                  job_state:
+                    type: string
+                    description: job_state can be&#58 PENDING, RUNNING, COMPLETED or FAILED
```

Comment:
Why you use `&#58` ?

Reply:
But good point for you, becuase it should be `&#58;` not `&#58`